app/src/controllers/log/actionCreators.js

```diff
@@ -0,0 +1,27 @@
+import {
+  FETCH_LOG_ENTRIES,
+  CHANGE_ACTIVE_LOG_ITEM,
+  SET_LOG_ITEM_TO_GET_HISTORY,
+} from './constants';
+
+export const fetchLogEntriesAction = () => ({
+  type: FETCH_LOG_ENTRIES,
+});
+
+export const setLogItemToGetHistoryAction = (logItemId) => (dispatch) => {
```

Comment:
We're moving from thunks to sagas, so please create saga instead.

Reply:
Done